ansible/provision.yaml

```diff
@@ -0,0 +1,106 @@
+---
+- name: Provision a k8s stack
+  hosts: localhost
+  connection: local
+  
+  # Include the aws account specific cluster variables
+  vars_files:
+    - vars/{{ environment_type }}-{{ aws_region }}.yaml
```

Comment:
At a second look, I'm not really sure on using the environment_type in the names of these files. I think we should rather reflect the topology of AWS, like per account.